docs/user-guides/algorithms_in_python.ipynb

```diff
@@ -122,7 +122,7 @@
     "    :return:\n",
     "    \"\"\"\n",
     "    # Iterate over the out edges of our node\n",
-    "    for edge_id in graph.out_edge_ids(node_id):\n",
+    "    for edge_id in graph.out_edge_ids_for_node(node_id):\n",
```

Comment:
I very much agree. I cannot overload methods in numba right now because it would take extra work and another week at least. This will change back when I can: KAT-4362.